src/Queue/Connectors/RabbitMQConnector.php

```diff
@@ -95,6 +96,7 @@ protected function createConnection(array $config): AbstractConnection
      */
     protected function createQueue(
         string $worker,
+        string $customWorkerType,
```

Comment:
Hi @M-Porter 
I some kind of confused
- as for me this is a minor change (only internal api has changed, protected method) and this can be released as minor version 
- the library by self has recently released breaking changes as minor version , by example https://github.com/vyuldashev/laravel-queue-rabbitmq/releases/tag/v13.1.0

Reply:
To be clear: v13.1.0 was a breaking minor version by the laravel library itself. We had to no choice but to follow suite as doing `composer update` for a patch version would have broken so to mitigate the problem created by laravel that was the only solution.

Yes, it's protected, but nothing keeps you from writing your own connector which others do and this would break `parent::createQueue()` calls.